cfme/utils/appliance/implementations/ui.py

```diff
@@ -54,6 +54,18 @@ def get_rails_error(self):
 
 class MiqBrowserPlugin(DefaultPlugin):
     ENSURE_PAGE_SAFE = jsmin('''\
+        try {
+            var eventNotificationsService = angular.element('#notification-app')
+                .injector().get('eventNotifications');
+            eventNotificationsService.clearAll(
+                ManageIQ.angular.eventNotificationsData.state.groups[0]
+            );
+            eventNotificationsService.clearAll(
+                ManageIQ.angular.eventNotificationsData.state.groups[1]
```

Comment:
So just to clarify, why 0 and 1? Can't we iterate over all? Any speciifc reason?

Reply:
events and tasks
I just felt iterating added another level of complexity we didn't need at this stage

Reply:
sure, just maybe add a comment that clarifies this.